apps/web/pages/mmb/other.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import * as React from 'react';
+import { ManageOther } from '@kiwicom/margarita-core';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Layout from '../../components/Layout';
+
+const ManageOtherScreen = () => (
+  <Layout>
+    <View style={styles.wrapper}>
+      <ManageOther />
+    </View>
+  </Layout>
+);
+
+const styles = StyleSheet.create({
+  wrapper: {
```

Comment:
Should we consider moving this to layout component?

Reply:
I moved it to a local Layout component; it seems specific to MMB views for now.